cdxev/validator/validate.py

```diff
@@ -100,7 +100,24 @@ def validate_sbom(
         )
         for error in sorted(v.iter_errors(sbom), key=str):
             try:
-                if len(error.absolute_path) > 3:
+                if error.validator == "required" and error.validator_value == [
+                    "this_is_an_externally_described_component"
```

Comment:
From an implementation point of view: IMO it would be easier to just validate the component and then check here if the component includes an external reference pointing to a SBOM, no? Having this workaround/check in this one file here, would reduce the risk of doing wrong changes in the schema.

Reply:
After internal discussions, you convinced me for not making this kind of implementation change, as we then would need to differentiate between _custom_ and _stock_ requirements, like e.g. name being still mandatory, so it's not as simple as just ignoring _all_ validation errors if a component has a reference to an external bom. It would increase the complexity and also requiring to differentiate between different cdx versions.